test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -1,5 +1,7 @@
 using namespace System.Diagnostics
 
+Import-Module $PSScriptRoot\..\Common\Test.Helpers.psm1
```

Comment:
My thinking:
say we have the module structure:
```
PowerShell
  | test
    | Modules
      | TestHelpers
        | TestHelpers.psd1
        | TestHelpers.psm1
```

Then, in `Start-PSPester`, ensure `PowerShell\test\Modules` is in `$env:PSModulePath`.
As long as the commands are exported properly, then the module will be auto-loaded.
Developers not wanting to run `Start-PSPester` would just ensure they have their module path set correctly.